src/test/java/blang/Examples.xtend

```diff
@@ -4,13 +4,15 @@ import blang.core.Model
 import blang.distributions.Bernoulli
 import blang.distributions.Beta
 import blang.distributions.Binomial
+import blang.distributions.BetaBinomial
```

Comment:
There are more steps needed to actually run the tests on these two new distributions (do a search in that page for another one that was added recently for example.)